ansible/books/ceph.yaml

```diff
@@ -191,6 +112,171 @@
         state: present
       when: '"rgw" in task_roles'
 
+- name: Ceph - Generate cluster keys and maps
+  hosts: server01
```

Comment:
We can't do that, when deploying on actual servers, we can't assume the name of the servers.